MoEPluginBase/MoEPluginCoordinator.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  MoEPluginCoordinator.swift
+//  MoEPluginBase
+//
+//  Created by Rakshitha on 23/06/22.
+//
+
+import Foundation
+import MoEngageSDK
+
+class MoEPluginCoordinator {
+    
+    var pluginControllers = [String: Any]()
```

Comment:
@RakshithaAcharya Let's use right access specifiers here too. Also Please make sure it is handled in all the classes added newly in this PR. 
We need to mark the class as final keyword to avoid the subclassing of any newly added classes.

Reply:
@deepa-iOS-SDK done